public/faq.html

```diff
@@ -10,6 +10,45 @@
             <title>DWP Analytics</title>
 <!--            <link href="/static/css/3.b6a8c48d.chunk.css" rel="stylesheet">-->
 <!--            <link href="/static/css/main.317a0afc.chunk.css" rel="stylesheet">-->
+            <script>
+                let showAnswer = (event) => {
+                    if (event.target.tagName === "IMG"){
+                    var expandButton = event.target.parentNode
+                  } else {
+                    var expandButton = event.target
+                  }
```

Comment:
Not a fan of `var`, it doesn't respect block scope (declaration gets hoisted to the top of the function or global scope if not in a function).
Can be simplified to `const expandButton = event.target.tagName === "IMG" ? event.target.parentNode : evenet.target`